index.js

```diff
@@ -522,6 +530,32 @@
         });
       });
 
+  //# reduce :: Foldable f => (a -> b -> a) -> a -> f b -> a
+  //.
+  //. Returns a single value by iterating through a list, successively
+  //. calling an iterator function and passing it an accumulator value and
+  //. the current value from the list, and then passing the result to the
+  //. next call.
```

Comment:
How about something like this?

_Takes a binary function, an initial value, and a [Foldable][1], and applies the function to the initial value and the Foldable's first value, then applies the function to the result of the previous application and the Foldable's second value. Repeats this process until each of the Foldable's values has been used. Returns the initial value if the Foldable is empty; the result of the final application otherwise._


[1]: https://github.com/fantasyland/fantasy-land#foldable